src/theme.ts

```diff
@@ -62,7 +61,7 @@ export const theme = {
     greens: [
       'hsl(175, 60%, 99%)',
       'hsl(175, 60%, 97%)',
-      'hsl(175, 40%, 72%)',
+      'hsl(175, 40%, 88%)',
       'hsl(175, 34%, 64%)',
       'hsl(175, 100%, 33%)',
       'hsl(175, 62%, 40%)',
```

Comment:
Green colors are not the right one.
I generated a new configuration based on Zeplin:
```
    green: 'hsl(129, 93%, 33%)',
    greens: [
      'hsl(129, 60%, 99%)',
      'hsl(129, 60%, 97%)',
      'hsl(129, 40%, 88%)',
      'hsl(129, 34%, 64%)',
      'hsl(129, 93%, 33%)',
      'hsl(129, 62%, 30%)',
      'hsl(129, 60%, 20%)',
      'hsl(129, 60%, 12%)',
      'hsl(129, 60%, 10%)',
    ],
```

Reply:
I have also seen that we have a lightBlue color.
Here is a configuration for it:
```
lightBlue: 'hsl(192, 69%, 45%)',
    lightBlues: [
      'hsl(192, 82%, 99%)',
      'hsl(192, 80%, 97%)',
      'hsl(192, 78%, 90%)',
      'hsl(192, 76%, 68%)',
      'hsl(192, 69%, 45%)',
      'hsl(192, 82%, 45%)',
      'hsl(192, 67%, 35%)',
      'hsl(192, 67%, 12%)',
      'hsl(192, 67%, 10%)',
    ],
```

Reply:
I made a sketch to make it clearest:
<img width="466" alt="Screen Shot 2019-12-11 at 11 30 13" src="https://user-images.githubusercontent.com/1550192/70613825-9fd45e80-1c09-11ea-92a8-b863a04cfdc5.png">